iam-login-service/src/main/resources/application.yml

```diff
@@ -154,6 +154,9 @@ iam:
     oidc-issuer: ${IAM_REGISTRATION_OIDC_ISSUER:https://example.org}
     saml-entity-id: ${IAM_REGISTRATION_SAML_ENTITY_ID:urn:example}
     add-nickname-as-attribute: ${IAM_ADD_NICKNAME_AS_ATTRIBUTE:false}
+    fields:
+      notes:
+        field-behaviour: ${IAM_REGISTRATION_NOTES_FIELD_BEHAVIOUR:mandatory}
```

Comment:
If we add this env variable we should add also the others: IAM_REGISTRATION_GIVEN_NAME_BEHAVIOUR, etc. I think we can avoid adding these lines and keep the default value as MANDATORY on this line https://github.com/indigo-iam/iam/pull/782/files#diff-eb3a40a655172a85a4ebaccf2dba240d9b4f294761e61a5631af9892aa95a49fR206